configs/skl_with_context_config.json

```diff
@@ -0,0 +1,77 @@
+{
+    "common": {
+        "lib": ["sklearn"],
```

Comment:
How stock or intel version of sk is specified for this config?

Reply:
Do we need to launch the stock sk in this config? Maybe just add flag `no-intel-optimized` before config run? I think it's better to separate patched and unpatched sk launches

Reply:
Ok - probably its better to skip `device != None` branches for stock sklearn

Reply:
Need to add support to skip these cases?